lib/Parse/ParseRequests.cpp

```diff
@@ -364,7 +366,8 @@ SourceFileParsingResult parseSourceFile(SourceFile &SF) {
       break;
     }
 
-    case GeneratedSourceInfo::MemberAttributeMacroExpansion: {
+    case GeneratedSourceInfo::MemberAttributeMacroExpansion:
+    case GeneratedSourceInfo::Attribute: {
       parser.parseExpandedAttributeList(items);
```

Comment:
Oh apparently, `swift_attr` accepts modifiers OR attributes, but `parseExpandedAttributeList` only accepts attributes? 🤔

Reply:
Yeah, and we have some things that are permitted but are neither attributes nor modifiers, which is odd. I'm going to have to think about this refactoring more.